installation-bundle/src/Command/LockCommand.php

```diff
@@ -46,12 +46,6 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        if (file_exists($this->lockFile)) {
```

Comment:
We could still return early if the lock file already exists. Maybe even add a different message like @aschempp did here?

https://github.com/contao/contao/blob/dba2cf227e062bfc06e6b8c1a2c52c8df4fbb21b/manager-bundle/src/Command/MaintenanceModeCommand.php#L115-L124

Reply:
I thought about this as well. But what if the lock file is a) compromised or b) we increase the lock count to (e.g.) 5 in a future Contao version? Then you won't be able to lock the Install Tool with the command at all.

Reply:
I agree with @richardhj. The `contao:maintenance-mode enable` command always writes the maintenance template again, even if the file exists. This command should do the same, we don't know if the existing file contains the correct data.

Reply:
What about adding different messages as described above?

Reply:
A message that says it was already enabled? We would need to check the file content for that, and we alao don't do that for maintenance mode 🤷‍♂️